network-api/networkapi/wagtailpages/models.py

```diff
@@ -58,7 +58,7 @@ def titlecase(s):
 base_fields = [field for field in [
     ('paragraph', blocks.RichTextBlock(
         features=[
-            'bold', 'italic',
+            'bold', 'italic', 'bodylarge',
```

Comment:
Should we rename this to `body_large` to align with our naming convention?

Reply:
I think it probably makes more sense to call this `large` rather than `bodylarge`, thinking about it some more